letta/main.py

```diff
@@ -361,8 +361,10 @@ def run_agent_loop(
         skip_next_user_input = False
 
         def process_agent_step(user_message, no_verify):
+            # TODO(charles): update to use agent.step() instead of inner_step()
```

Comment:
This needs to be done eventually to support non-MemGPT agents on the Letta CLI / `letta run`